### PR TITLE
Multiple Repos: start writing them to the DB

### PR DESCRIPTION
**Pull request by danielbeardsley**
The first step in multi-repo support is to add the repo columns
and start writing to them.

* Migration to add the repo column to all the tables
   * (not part of the PK yet)
* Add the "repo" property to the models in all the right places
* Start writing to the new db column for all the tables

Note: this doesn't mean we support multiple repos yet; just that the
repo data is now recorded.

Connect #62

### CODE REVIEW

**Comment by danielbeardsley** [comment]
I've got this running, refreshing, saving info to the DB, and listening to GH webhooks.

I'm gonna say QA 👍

File: models/pull.js [hunk 1]
```diff
@@ -189,12 +158,50 @@ Pull.parseBody = function(body) {
...
+      state: data.state,
+      title: data.title,
+      body: data.body,
+      created_at: utils.fromDateString(data.created_at),
+      updated_at: utils.fromDateString(data.updated_at),
+      closed_at: utils.fromDateString(data.closed_at),
+      merged_at: utils.fromDateString(data.merged_at),
+      difficulty: data.difficulty,
+      milestone: {
+         title: data.milestone && data.milestone.title,
```
Comment (MooreGuy) [inline models/pull.js]: TIL that you can do this

**Comment by MooreGuy** [comment]
CR :+1:

**Comment by djmetzle** [comment]
CR 👍

**Pull request merged**